frame/fast-unstake/src/lib.rs

```diff
@@ -381,9 +390,12 @@ pub mod pallet {
 					num_slashing_spans,
 				);
 
+				T::DepositCurrency::unreserve(&stash, deposit.into());
```

Comment:
please add `defensive` checks on the return value of this, making sure the remainder is zero.